c_sharp/Snpe_EP/vgg16_image_classification/README.md

```diff
@@ -1,83 +1,51 @@
 # Image classification with VGG16 in C# using SNPE Execution Provider
-1.  This image classification sample uses the VGG16 model from [SNPE turtorial](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_onnx.html) with DLC format. Wrap it as a custom node in an ONNX model to inference with SNPE Execution Provider.
 
-2.  The sample uses the Onnxruntime SNPE Execution Provider to run inference on various Qualcomm devices like Qualcomm CPU, DSP, etc. It supports Windows ARM64.
-
-# Prerequisites
-1. Setup a Linux environment by [WSL2](https://learn.microsoft.com/en-us/windows/wsl/)
-2. Download SNPE SDK from Qualcomm's developer site [here](https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk/windows-on-snapdragon). Developer needs to request access to the package. SNPE v1.61.46 and v1.61.48 are qualified so far.
-
-3. Setup SNPE on the Linux environment (WSL2). Setup environment for the tutorial. Follow the Tutorials and Examples for [ONNX VGG](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_onnx.html)
-4. Get the model
-    Run command below to get the VGG16 Onnx model.
-    ```
-    cd $SNPE_ROOT/models/VGG
-    wget https://s3.amazonaws.com/onnx-model-zoo/vgg/vgg16/vgg16.onnx
-    ```
-
-    Offload the Softmax from post-processing to model inference (on NPU). Copy add_softmax.py to $SNPE_ROOT/models/VGG/onnx folder and run it, to apply Softmax node to model output.
-
-    Follow step 3 ~ 5 to generate the DLC file vgg16.dlc. 
-	
-	Runn command below to generate the quantized DLC file vgg16_q.dlc.
+This sample shows how to use ONNX Runtime with SNPE execution provider to run models optimized for Qualcomm silicon on Windows Arm64 devices. It uses the VGG16 model from [Qualcomm's tutorial](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_onnx.html) and turns it into a SNPE-optiimized ONNX model that can be inferenced with ONNX Runtime with SNPE execution provider.
+
+# Install Qualcomm SDK for Windows
+Download the Qualcomm Neural Processing SDK for Windows on Snapdragon from [Qualcomm's developer site](https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk/windows-on-snapdragon).
+* You currently need to request access to the packages.
+* SNPE v1.61.46 and v1.61.48 have been qualified with ONNX Runtime so far.
+
+# Prepare the SNPE-optimized ONNX file 
+Currently the Qualcomm model tools are only runnable in a Linux environment even though their output will be used in Windows.
+1. Setup a Linux environment with [WSL2](https://learn.microsoft.com/en-us/windows/wsl/)
+2. Download and install the Qualcomm SNPE SDK in the WSL2 environment.

```

Comment:
should we clarify that they need to download the Linux SDK (and provide direct link) for the model tool/examples ?
i.e. the models directory only exists in the Linux SDK